libs/langchain-baidu-qianfan/src/chat_models.ts

```diff
@@ -72,13 +70,13 @@ interface ChatCompletionResponse {
  */
 declare interface BaiduQianfanChatInput {
   /**
-   * Model name to use. Available options are: ERNIE-Bot, ERNIE-Bot-turbo, ERNIE-Bot-4
+   * Model name to use. Available options are: ERNIE-Bot, ERNIE-Lite-8K, ERNIE-Bot-4
    * Alias for `model`
-   * @default "ERNIE-Bot-turbo"
+   * @default "ERNIE-Lite-8K"
```

Comment:
nit, revert back to original
```suggestion
   * @default "ERNIE-Bot-turbo"
```